.github/workflows/backend-workflow.yml

```diff
@@ -0,0 +1,26 @@
+name: Backend CI
+on: 
+  push:
+    branches-ignore:
+      - 'frontend/**' # Don't run on any frontend branches

```

Comment:
Rather than doing `branches: - 'backend/**`, we just exclude frontend branches from running it. This way pushes to the master or branch or small direct-to-master fixes (such as this one) have the actions run on their commits.